src/plugins/data_views/server/ui_settings.ts

```diff
@@ -8,9 +8,27 @@
 
 import { i18n } from '@kbn/i18n';
 import { schema } from '@kbn/config-schema';
+import type { UiSettingsParams } from '@kbn/core/server';
+import { DATA_VIEWS_FIELDS_EXCLUDED_TIERS } from '../common/constants';
 import { DEFAULT_FIELD_CACHE_FRESHNESS } from './constants';
 
+export const dataTiersUiSettingsConfig: Record<string, UiSettingsParams> = {
+  [DATA_VIEWS_FIELDS_EXCLUDED_TIERS]: {
+    name: i18n.translate('dataViews.advancedSettings.dataTiersName', {
+      defaultMessage: 'Data tiers excluded from field requests',
+    }),
+    value: '',
+    type: 'string',
+    description: i18n.translate('dataViews.advancedSettings.dataTiersText', {
+      defaultMessage:
+        'Exclude fields from specified tiers (such as data_frozen) for faster performance.',
+    }),
+    schema: schema.string(),
+  },
+};
+
 export const cacheMaxAge = {
+  // to do probably use variable from constants
```

Comment:
@mattkime I guess this is a to do , oder a to be removed?

Reply:
Its a good change to make but separate from the concerns in this pr. I'll remove this additional comment.